### ISSUE TITLE
[Bug]: Display 'case' instead of 'headphone' in battery notice when battery is lower than 25%

### ISSUE THREAD

**Issue by Iliannnn** [steam3d/MagicPods-Windows#398]
The headphones (AirPods) itself aren't lower than 25% but the case is in this case, it would be better to mention 'case' instead of 'headphones':
![image](https://github.com/steam3d/MagicPods-Windows/assets/72648901/dfd1dec0-5e0f-4f81-bb87-1520ffca51f3)

**Comment by steam3d** [comment]
Hi, it's expected behavior. The MagicPods check left, right, and case battery to detect which battery has low level value

**Comment by Iliannnn** [comment]
Yeah, then this is a feature request to display the correct part accordingly.
